commands/faas.go

```diff
@@ -42,11 +50,20 @@ func init() {
 
 // Execute TODO
 func Execute(customArgs []string) {
+	checkAndSetDefaultYaml()
+
 	faasCmd.SilenceUsage = true
 	faasCmd.SetArgs(customArgs[1:])
 	faasCmd.Execute()
 }
 
+func checkAndSetDefaultYaml() {
```

Comment:
I want to get the fix in, think this is going to be a good addition. This method feels hacky to do this to a global variable. @johnmccabe any other suggestions?